bitnami/minio/templates/provisioning-job.yaml

```diff
@@ -60,6 +60,9 @@ spec:
                 --timeout=120 \
                 {{ .Values.service.ports.api | int64 }};
               echo "Minio is available";
+          {{- if .Values.provisioning.resources }}
```

Comment:
I think you should update values.yaml to document this

for example: https://github.com/bitnami/charts/blob/master/bitnami/kafka/values.yaml#L766

Reply:
It is already in the value file: https://github.com/bitnami/charts/blob/master/bitnami/minio/values.yaml#L242